docs/Introduction/tutorials/advanced-tutorial.md

```diff
@@ -25,19 +29,26 @@ In this guide, you will learn how to request data from a public API in a smart c
 
 **Table of Contents**
 
-+ [Overview](#overview)
-+ [1. How does the request and receive cycle work for API calls?](#1-how-does-the-request-and-receive-cycle-work-for-api-calls)
-+ [2. What are initiators?](#2-what-are-initiators)
-+ [3. What are Adapters?](#3-what-are-adapters)
-+ [4. How can I use Adapters in my own contract?](#4-how-can-i-use-adapters-in-my-own-contract)
-+ [5. How do I deploy to testnet?](#5-how-do-i-deploy-to-testnet)
-+ [6. Further Reading](#6-further-reading)
+- [Overview](#overview)
+- [1. How does the request and receive cycle work for API calls?](#1-how-does-the-request-and-receive-cycle-work-for-api-calls)
+- [2. What are initiators?](#2-what-are-initiators)
```

Comment:
I think we better talk about _Jobs_ rather than [_Initiators_](https://documentation-git-dot-delimited-api-calls-chainlinklabs.vercel.app/docs/jobs/#what-is-a-job) since we migrated from JSON to TOML. and point all the references to the right jobs. For instance:

[RunLog](/docs/initiators/#runlog) sould be replaced by [Direct Request Job](/docs/jobs/types/direct-request/)

what do you think? I can take care of it if you want